.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - "*"
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  ci:
+    name: ${{ matrix.task.name}} - ${{ matrix.python.name }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - name: CPython 2.7
+            tox: py27
+            action: 2.7
+          - name: CPython 3.9
+            tox: py39
+            action: 3.9
+          - name: PyPy 2
+            tox: pypy2
+            action: pypy2
+          - name: PyPy 3
+            tox: pypy3
+            action: pypy3
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up ${{ matrix.python.name }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python.action }}
+
+    - name: Run the action
```

Comment:
Can we just have `- uses: ./` here so that we can see the default name?

Reply:
I actually set the name here because it had a terrible name, just `./` if I remember correctly.  But yeah, making it not default to that would be the obviously better option.  I'll see what I can do there.

Reply:
https://github.com/actions/checkout/blob/5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f/action.yml#L1
```yaml
name: 'Checkout'
```
Yet we get:
![image](https://user-images.githubusercontent.com/543719/100481575-7a628e00-30c2-11eb-937d-cc11cdd62bc2.png)

I'm not sure you get to specify the name?  I think it just defaults to the action ID?  So the local run would just be `Run ./` and in other workflows it would be `Run twisted/python-info-action`.  In which case it may as well have this descriptive name.  Are you familiar with any info to the contrary?  I did also search through https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-composite-run-steps-action.

Reply:
thanks for the info. then is best to just use the custom name as you already did.
I don't know how the name is defined.